test/ecto/query/subquery_test.exs

```diff
@@ -563,14 +563,5 @@ defmodule Ecto.Query.SubqueryTest do
 
       assert Macro.to_string(planned_q.from.source.query.select.expr) == "%{id: ^0, text: ^1}"
```

Comment:
only remaining question is that this test above ^ was added as part of the PR that initially produced this check, but this test does pass. I left this test, but can remove it if desired.